lib/install/management/store_files.go

```diff
@@ -40,51 +38,47 @@ const (
 	volumeRoot = "volumes"
 )
 
-func (d *Dispatcher) DeleteStores(vchVM *vm.VirtualMachine, conf *config.VirtualContainerHostConfigSpec) error {
-	defer trace.End(trace.Begin(""))
+func (d *Dispatcher) deleteImages(conf *config.VirtualContainerHostConfigSpec) error {
+	var emptyImages bool
+	var errs []string
 
-	ds := d.session.Datastore
+	for _, imageDir := range conf.ImageStores {
+		imageDSes, err := d.session.Finder.DatastoreList(d.ctx, imageDir.Host)
+		if err != nil {
+			errs = append(errs, err.Error())
+			continue
+		}
 
-	p, err := d.getVCHRootDir(vchVM) // p would be path but there's an imported package called path
-	if err != nil {
-		return err
-	}
+		if len(imageDSes) != 1 {
```

Comment:
I don't think there's anything the user can do to address this as we're pulling the input from an existing config.
Will -force let the user progress past this error?